alpine-rails-buildkite/etc/sv/buildkite

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+export HOME=/var/buildkite
+export GEM_HOME=${HOME}/.gem
+export BUILDKITE_AGENT_VERSION=edge
+export BUILDKITE_BUILD_PATH=/var/buildkite/builds
+export BUILDKITE_HOOKS_PATH=/var/buildkite/hooks
+export BUILDKITE_BOOTSTRAP_SCRIPT_PATH=/usr/lib/buildkite/bootstrap.sh
+
+exec chpst -u buildkite -- /usr/bin/buildkite-agent start
```

Comment:
Its in the path, but I simply assumed that `chpst` does not rely on past and just like other utilities prefers an absolute binary reference.